penciledgedev/Cimee-Mobile
# Ft user schedule meeting

### What does this PR do??
- Users should be able to create a meeting (scheduled or for later)

### How should this be tested manually?
- Clone this Repo
- Checkout to this branch
- run `expo install` to install all dependencies 
- Start the metro bundler by running ` expo start `
- Tunnel the application on android using android virtual device on android studio or on ios using xcode
- See #2 to login to the application
- Navigate to the host screen by clicking on the floating action button to host a meeting
- Enter the valid credentials 
- If successful it will direct you to a start meeting screen

### Any other Context you want to add?
- Not applicable
